api4/schema.graphqls

```diff
@@ -85,24 +90,40 @@ type ChannelMember {
 type User {
 	id: String!
 	username: String!
+	# deliberately omitting password
 	email: String!
+	authData: String
```

Comment:
AuthData probably shouldn't be included. I notice it exists in the TS types, but it's always sanitized out before sending to users, similar to MfaSecret and Password.

https://github.com/mattermost/mattermost-server/blob/master/model/user.go#L618